src/QsCompiler/LlvmBindings/Values/GlobalAlias.cs

```diff
@@ -17,7 +17,6 @@ public class GlobalAlias
         public Constant Aliasee
         {
             get => this.IndirectSymbol!;
-            set => this.IndirectSymbol = value;
```

Comment:
Was this an intentional removal? Just curious why it cannot (or maybe should not) be supported.

Reply:
Good catch! Yes this was accidental. I ripped out the setter methods from `IOperandCollection` since metadata operands cannot be set via the LLVM C API, but didn't notice there were other implementations. I'll add these back and probably split `IOperandCollection` into `IReadableOperandCollection` and `IWritableOperandCollection` (like C#'s `IReadOnlyList`).

Reply:
Should be all set! I actually didn't need to do anything other than add back the removed lines since `User.Operands` is actually exposed as the concrete type (`ValueOperandListCollection`), i.e. not `IOperandCollection`.